RELEASE_NOTES.md

```diff
@@ -6,6 +6,14 @@ MSA Release Notes
 * Upgrade SAML libs version so that the MSA can support the ECDSA and RSASSA-PSS signing algorithms to fulfil the eIDAS cryptographic requirements
 as laid out in the "eIDAS - Cryptographic requirements for the Interoperability Framework". 
 * Upgrade to Dropwizard 1.3.5 plus updated various other libs, specifically: ida_utils, ida_test_utils, hub_saml, saml_libs_version.
```

Comment:
@galund @phantas - could you guys have a look at rephrasing the above?

Reply:
Hi @bravegrape, I'm not clear who these notes are really intended for (technical person or not? What would they use them to decide to do / not do?). The library updates are a consequence of the Dropwizard upgrade, but if the end user probably won't care, then it may be sufficient to write "Upgrade to Dropwizard 1.3.5" - alternatively if they would need more detail then we should look in the release notes for each of those libs. Or we could go with less detail, and just say "Security fixes".

Reply:
Hi @galund, as far as I understand the release notes are mainly intended for RPs. The users would be technical people, but as you say, they probably won't care which libraries were updated as a consequence of the Dropwizard update. 

I agree, "Upgrade to Dropwizard 1.3.5" would suffice 👌